src/main/java/org/folio/service/budget/RolloverBudgetExpenseClassTotalsService.java

```diff
@@ -55,25 +55,32 @@ private List<BudgetExpenseClassTotal> buildBudgetExpenseClassesTotals(List<Expen
     Map<ExpenseClass, List<Transaction>> groupedByExpenseClass = expenseClasses.stream()
       .collect(toMap(Function.identity(), expenseClass -> groupedByExpenseClassId.getOrDefault(expenseClass.getId(), Collections.emptyList())));
 
-    return buildBudgetExpenseClassesTotals(groupedByExpenseClass, totalExpended);
+    return buildBudgetExpenseClassesTotals(groupedByExpenseClass, totalExpended, totalCredited);
   }
 
   private double getBudgetTotalExpended(LedgerFiscalYearRolloverBudget budget) {
-    BigDecimal totalExpended = BigDecimal.valueOf(budget.getExpenditures()).add(BigDecimal.valueOf(budget.getOverExpended()));
-    return totalExpended.doubleValue();
+    return budget.getExpenditures();
```

Comment:
As in the mod-finance PR for MODFIN-372, this can be simplified by removing the method.

Reply:
Thanks fixed this